### PR TITLE
Turn on edge bits

### PR DESCRIPTION
**Pull request by esheldon**
set the EDGE bit for a border around the image

### CODE REVIEW

**Comment by esheldon** [comment]
Note it turns out this does not address #27

**Comment by esheldon** [comment]
I added a test; what do you have in mind?

File: descwl_shear_sims/simple_sim.py [hunk 1]
```diff
@@ -27,10 +34,6 @@
     'wldeblend': {'ngals_factor': 1.0},
 }
 
-# default mask bits from the stack
-BAD_COLUMN = np.int64(2**0)
-COSMIC_RAY = np.int64(2**3)
-
 
 # double check they match the stack
 def _check_bits_against_stack():
```
Comment (beckermr) [inline descwl_shear_sims/simple_sim.py]: This function should go in the lsst_bits.py module.

**Comment by beckermr** [comment]
This test needs updating: https://github.com/LSSTDESC/descwl-shear-sims/blob/master/descwl_shear_sims/tests/test_lsst_mask_bits.py

File: descwl_shear_sims/simple_sim.py [hunk 2]
```diff
@@ -90,6 +68,8 @@ class Sim(object):
     buff : int, optional
         The width of the buffer region in the coadd image where no objects are
         drawn. Default is 50.
+    edge_width:  int
+        Width of boundary to be marked as EDGE in the bitmask
```
Comment (beckermr) [inline descwl_shear_sims/simple_sim.py]: I'd be nice to have the default in the doc string.

**Pull request merged**